module/account/src/Infrastructure/Handler/Role/UpdateRoleCommandHandler.php

```diff
@@ -29,7 +28,16 @@ public function __construct(RoleRepositoryInterface $repository)
     public function __invoke(UpdateRoleCommand $command): void
     {
         $role = $this->repository->load($command->getId());
-        Assert::isInstanceOf($role, Role::class, sprintf('Can\'t find Role with id %s', $command->getId()));
+
+        if (!$role instanceof Role) {
+            throw new \LogicException(
+                sprintf(
+                    'Expected an instance of %s. %s received.',
+                    Role::class,
+                    get_debug_type($role)
```

Comment:
This function has been added to PHP 8. This project currently using 7.4
https://wiki.php.net/rfc/get_debug_type

Reply:
@piotrkreft czy ja dobrze rozumiem, że jeżeli widze w symfony.locku, że jest instalowany symfony/polyfill-php80 to on dostarcza tą funkcje?

Reply:
Yup, it's part of the installed polyfill. Safe to use and FC